src/docs/customize/component-customization.mdx

```diff
@@ -0,0 +1,184 @@
+---
+name: Component Customization
+menu: Customize
+route: /customize/component-customization
+---
+
+# Component Customization
+
+## Global props
```

Comment:
Please stay consistent in capitalizing: _Global Props_, also see other headings below.